app/src/main/java/org/systers/mentorship/view/activities/SendRequestActivity.kt

```diff
@@ -36,14 +43,33 @@ class SendRequestActivity: BaseActivity() {
         val currentUserId = getAuthTokenPayload().identity
         setObservables()
         populateView(otherUserName, otherUserId, currentUserId)
+
+        myCalendar = Calendar.getInstance()
+        var date : DatePickerDialog.OnDateSetListener = object : DatePickerDialog.OnDateSetListener{
+            override fun onDateSet(view: DatePicker?, year: Int, month: Int, dayOfMonth: Int) {
+                myCalendar.set(Calendar.YEAR , year)
+                myCalendar.set(Calendar.MONTH  , month)
+                myCalendar.set(Calendar.DAY_OF_MONTH  , dayOfMonth)
+                updateEditText()
+            }
+        }
+        calendar_icon.setOnClickListener {
+          DatePickerDialog(this , date ,
+                  myCalendar.get(Calendar.YEAR) ,
+                  myCalendar.get(Calendar.MONTH) ,
+                  myCalendar.get(Calendar.DAY_OF_MONTH)).show()
+      }
+    }
+    private fun updateEditText() {
```

Comment:
Can you change the name of this function to be more descriptive? Because we have more than one edit text. As an example you could call it `updateEndDateEditText()`

Reply:
ok